### ISSUE TITLE
Implement Windows Service

### ISSUE THREAD

**Issue by timotheux** [halildurmus/win32#230]
Will you be willing to implement the ability to run a dart app as Windows Service?

I believe this is an area dart/win32 is made for. Dart already is an excellent console application on Windows.

Being able to run it as a Windows service will allow a lot of programmers port existing or design new programs/tools in the dart language.

Keepup the good work!

**Comment by timsneath** [comment]
Not sure whether we'll run into threading issues, but it's an interesting idea, for sure. 

Sample documentation here: https://docs.microsoft.com/en-us/windows/win32/services/the-complete-service-sample

**Comment by timotheux** [comment]
Yes, it's an interesting idea with real world use cases right now and it will go on to have a high adoption rate.

Have a look at this solution from Tuppeny, I think this brings us closer to a global solution:
https://stackoverflow.com/a/63744433

Cheers

**Comment by timotheux** [comment]
@timsneath just checked out your named pipe example. Good Job! Once again.

Any progress on the Windows Service front?

Cheers

**Comment by timsneath** [comment]
It's not an area I'm planning to work on right now, but if others are interested I'm very willing to accept contributions.

**Comment by biloo-dev** [comment]
i'm very interested for this please any update !!

**Comment by timsneath** [comment]
It's not an area I'm planning to work on right now, but if others are interested I'm very willing to accept contributions.

**Comment by au-top** [comment]
look at package https://pub.dev/packages/dart_windows_service_support
